src/OrbitGl/TriangleToggle.cpp

```diff
@@ -34,31 +34,32 @@ void TriangleToggle::DoDraw(PrimitiveAssembler& primitive_assembler, TextRendere
   Color color = is_collapsible_ ? kWhite : kGrey;
 
   // Draw triangle.
-  static float half_sqrt_three = 0.5f * sqrtf(3.f);
-  float half_triangle_base_width = 0.5f * GetWidth();
-  float half_triangle_height = half_sqrt_three * 0.5f * GetHeight();
-
   const Vec2 pos = GetPos();
   if (!picking) {
-    Vec2 position(pos[0], pos[1]);
+    Vec2 midpoint = GetPos() + Vec2(GetWidth() / 2, GetHeight() / 2);
 
     Triangle triangle;
     if (is_collapsed_) {
-      triangle = Triangle(position + Vec2(-half_triangle_height, half_triangle_base_width),
-                          position + Vec2(-half_triangle_height, -half_triangle_base_width),
-                          position + Vec2(half_triangle_base_width, 0.f));
+      const float triangle_height = sqrtf(3.f) * GetWidth() * 0.5f;
```

Comment:
Maybe `constexpr float kRootOfThreeF = sqrtf(3.f)` ?

Reply:
or even `constexpr float kRootOfThreeFOverTwo = sqrtf(3.f) / 2`

... or `constexpr float cos30F = sqrtf(3.f) / 2`

Reply:
What do we gain from moving this into a constant? Because it's used in both if-clauses?

Reply:
> Because it's used in both if-clauses?

That's one thing. 

Another -- where does that value come from? E.g. if in your calculations you had `cos(30)`, I suggest to call it so.

Reply:
I've changed this according to the discussion above to use the same size in both collapsed and uncollapsed state, so it's no longer used twice.

As for the value itself comes from the simplest formula to calculate the height of a triangle with equal side length: `sqrt(3) * len / 2` - I can use something like `kRootOfThreeFOverTwo`, but I really don't see what that tells me more than the values themselves TBH.

Reply:
Disclaimer: my geometry is rusty, and it's just a personal preference. Feel free to disagree and keep it as is. 

`sqrt(3) * len / 2` actually comes from from `cos(60) * len` and naming the variable like `kCos60` helps readability IMO.

Further, from the line `const float triangle_height = sqrtf(3.f) * GetWidth() * 0.5f;` it's hard to to deduce that we work with an equilateral triangle (I couldn't). 

Perhaps, there should also be a free function like `HeightOfEqulateralTriangle(int side_length)`.

Reply:
Ok true. You can also get the formula from resolving `height^2 + (width / 2)^2 = width^2`, which is how I remembered this - but obviously, this is related to the calculation of cos, and `cos(30) == sqrt(3) / 2`.

So in conclusion: I'll add the constant, and we both spend probably way more time on this than we should have 😄